### PR TITLE
Molecule home

### PR DESCRIPTION
**Pull request by fuentesloic**

### CODE REVIEW

File: content/styles/01-atoms/buttons.css [hunk 1]
```diff
@@ -25,17 +25,38 @@ input[type=submit] {
...
-  border: .125rem solid var(--primary-color);
+.read::before {
+  margin-right: 2rem;
+  font-family: "fontawesome solid";
+  font-size: 2rem;
+  content: "\f518";
+}
+
+.sign {
+  background-color: var(--primary-sign-color);
```
Comment (vinyll) [inline content/styles/01-atoms/buttons.css]: Is primary-sign-color going to be used to something else than sign?
If yes, the name might be irrelevant,
If not, the variable might be useless and the colors should be hardcoded here

File: content/styles/02-molecules/hero.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+.hero h3 {
+  color: var(--white);
+}
+
+.hero h2 {
+  font-size: 8rem;
+  margin-bottom: 0;
+}
+
+.hero .button {
```
Comment (vinyll) [inline content/styles/02-molecules/hero.css]: Hum?

File: content/styles/02-molecules/preview.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+.preview a {
+  color: var(--black);
+}
+
+.preview a:hover figure::before {
+  color: var(--secondary-color);
+  font-size: 8rem;
+  bottom: -0.5rem;
```
Comment (vinyll) [inline content/styles/02-molecules/preview.css]: I would put the bottom… on top ^^
bottom being a position property should be above appearance

**Review by vinyll — CHANGES_REQUESTED** [review]

File: content/styles/01-atoms/buttons.css (hunk 1 above)
Comment (fuentesloic) [reply content/styles/01-atoms/buttons.css]: this color (just a test for now) could be use for action with strong implication like publish, become a neomad, sign in, ... I know the name is pretty irrelevant but I don't have any idea.

It's an action, but I want something more strong to catch user eye et put the other regular link more quiet and smooth/soft.

Any idea on your side ??

File: content/styles/02-molecules/preview.css (hunk 3 above)
Comment (fuentesloic) [reply content/styles/02-molecules/preview.css]: totally true

File: content/styles/02-molecules/hero.css (hunk 2 above)
Comment (fuentesloic) [reply content/styles/02-molecules/hero.css]: eyes of tiger!!! (but already fix when you was writtin this ^^)

File: content/styles/01-atoms/buttons.css (hunk 1 above)
Comment (vinyll) [reply content/styles/01-atoms/buttons.css]: Suggestions for brainstorming:
- `primary-highlight-color`
- `primary-action-color`
- `primary-strong-color`
- …

**Review by vinyll — CHANGES_REQUESTED** [review]
Are you really sure about this?
`strong` and `action`?

**Review by vinyll — APPROVED** [review]

**Pull request merged**